sql/CMakeLists.txt

```diff
@@ -34,47 +34,110 @@ set(EXT_SQL_EXTRA_FILES
   timescaledb--0.4.2--0.5.0.sql
   timescaledb--0.5.0--0.6.0.sql
   timescaledb--0.6.0--0.6.1.sql
-  timescaledb--0.6.1--0.7.0.sql
   timescaledb--0.6.1--0.7.1.sql
-  timescaledb--0.7.0--0.7.1.sql)
-
-if (WIN32)
-  # Make list of files into string of files separated by "+"
-  # to make Windows copy concatenate them
-  string(REPLACE ";" ";+" SQL_FILES_JOINED "${SQL_FILES}")
-  # Windows copy command requires backslashes for relative paths
-  string(REPLACE "/" "\\" EXT_SQL_UPDATE_PRE_FILE "${EXT_SQL_UPDATE_PRE_FILE}")
-  string(REPLACE "/" "\\" EXT_SQL_UPDATE_POST_FILE "${EXT_SQL_UPDATE_POST_FILE}")
-
-  set(CAT_SQL_FILE_CMD copy /B /y ${SQL_FILES_JOINED} "\"${CMAKE_CURRENT_BINARY_DIR}/${EXT_SQL_FILE}\"" >NUL)
-  set(CAT_SQL_UPDATE_FILE_CMD copy /B /y ${EXT_SQL_UPDATE_PRE_FILE} + ${SQL_FILES_JOINED} + ${EXT_SQL_UPDATE_POST_FILE} "\"${CMAKE_CURRENT_BINARY_DIR}/${EXT_SQL_UPDATE_FILE}\"" >NUL)
-else ()
-  set(CAT_SQL_FILE_CMD cat ${SQL_FILES} > ${CMAKE_CURRENT_BINARY_DIR}/${EXT_SQL_FILE})
-  set(CAT_SQL_UPDATE_FILE_CMD cat ${EXT_SQL_UPDATE_PRE_FILE} ${SQL_FILES} ${EXT_SQL_UPDATE_POST_FILE} > ${CMAKE_CURRENT_BINARY_DIR}/${EXT_SQL_UPDATE_FILE})
-endif ()
-
-# Command and target for the SQL file
-add_custom_command(
-  OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/${EXT_SQL_FILE}
-  DEPENDS ${SQL_FILES}
-  WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-  COMMAND ${CAT_SQL_FILE_CMD}
-  COMMENT "Generating ${EXT_SQL_FILE}")
+  )
 
+#TODO handle these
```

Comment:
what does this mean?

Reply:
I am not entirely sure what should happen in such "fixup" cases. Figured we could punt till later